packages/editor/src/services/EditorHelperState.ts

```diff
@@ -53,15 +50,6 @@ export const EditorHelperState = defineState({
     'gridHeight'
   ]),
   reactor: () => {
-    const [showGlbChildrenFlag] = useFeatureFlags([FeatureFlags.Studio.UI.Hierarchy.ShowGlbChildren])
-
-    useEffect(() => {
-      const showGlbChildren = getMutableState(EditorHelperState).showGlbChildren
```

Comment:
we can remove showGlbChildren in the EditorHelperState too, and any place it is used